src/js/schemas/service-schema/Volumes.js

```diff
@@ -38,6 +43,11 @@ const Volumes = {
     },
     dockerVolumes: {
       title: 'Docker Volumes',
+      description: (
+        <span>
+        Create a stateful application using Docker volumes. <a href="https://docs.docker.com/engine/tutorials/dockervolumes/">Learn more about Docker volumes.</a>
```

Comment:
We need to indent this once further and add `target="_blank"` to the external link.